docs/sources/setup-grafana/configure-security/configure-authentication/okta/index.md

```diff
@@ -8,61 +8,100 @@ weight: 1200
 
 # Configure Okta OAuth2 authentication
 
-> Only available in Grafana v7.0+
+The Okta authentication enables Grafana users to sign in by using an external Okta authorization server.
 
-The Okta authentication allows your Grafana users to log in by using an external Okta authorization server.
+## Before you begin
+
+To complete this guide, you need:
```

Comment:
@Mkrummz , this is the place where we can add prerequisites such as permissions the user must have or any knowledge we assume that they have. Do we have any such before you begin items to add?